knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -163,35 +155,26 @@ func (r *ReconcileKnativeServing) ensureCustomCertsConfigMap(instance *servingv1
 
 // Install Kourier Ingress Gateway
 func (r *ReconcileKnativeServing) installKourier(instance *servingv1alpha1.KnativeServing) error {
-	return kourier.Apply(instance, r.client)
-}
 
-// create wide-open networkpolicies for the knative components
-func (a *ReconcileKnativeServing) installNetworkPolicies(instance *servingv1alpha1.KnativeServing) error {
-	namespace := instance.GetNamespace()
-	log.Info("Installing Network Policies")
-	const path = "deploy/resources/networkpolicies.yaml"
-
-	manifest, err := mf.NewManifest(path, false, a.client)
-	if err != nil {
-		return err
-	}
-	transforms := []mf.Transformer{mf.InjectOwner(instance)}
-	if len(namespace) > 0 {
-		transforms = append(transforms, mf.InjectNamespace(namespace))
-	}
-	if err := manifest.Transform(transforms...); err != nil {
+	// Set kourier to ingress.class
+	refetched := &servingv1alpha1.KnativeServing{}
+	if err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: instance.Namespace, Name: instance.Name}, refetched); err != nil {
 		return err
 	}
-	if err := manifest.ApplyAll(); err != nil {
-		return err
+
+	if common.Configure(refetched, "network", "ingress.class", "kourier.ingress.networking.knative.dev") {
```

Comment:
As discussed, we shouldn't need this anymore.

Reply:
Thank you. Sorry, I missed to drop this.